src/frontend/utils/toggles.ts

```diff
@@ -16,4 +16,5 @@ export enum Toggle {
     /**
      * features-under-utvikling
      */
+    KAN_SE_INNVILGELSE_BARNETILSYN_V2 = 'sak.frontend.kan-se-innviligese-barnetilsnyn-v2',
```

Comment:
Burde vi ha en toggle samlet for både frontend og backend? 
Og når denne slåes på, så er vedtaksperioder påkrevd for både frontend og backend? 👀

Reply:
Så lenge vi bruker egne endepunkter for V2 i backend tenker jeg det holder med en frontend toggle. Da blir det på en måte skrudd på samtidig for begge. 

Det krever også at vedtaksperioder er påkrevd for backend siden det nye endepunktet ikke har noe kjennskap til stønadsperioder